src/test/java/teammates/test/cases/datatransfer/StudentProfileAttributesTest.java

```diff
@@ -87,7 +87,7 @@ private void testGetInvalidityInfoForValidProfileWithValues() {
         assertEquals(new ArrayList<String>(), validProfile.getInvalidityInfo());
     }
 
-    private void testGetInvalidtyInfoForValidProfileWithEmptyValues() {
+    private void testGetFailureInfoForValidProfileWithEmptyValues() {
```

Comment:
I believe the problem here is that invalidity is mispelt so it should be changed to `testGetInvalidityInfoForValidProfileWithEmptyValues`.